src/extension/extension.ts

```diff
@@ -50,21 +45,23 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 
         await loadConfiguration();
 
-        const dotnetVersion: semver.SemVer = await dotnet.getHostVersion();
-        const canEnableLanguageService: Boolean = dotnetVersion && (dotnetVersion.compare(supportedRuntimeVersion) >= 0);
+        const dotnetExecutablePath = await dotnet.acquireRuntime(context.extension.id, progress);
```

Comment:
`context.extension` was not available in engine version 1.39. No idea when exactly it appeared, but it is 100% there for 1.72+